accesstoken2/src/types/errors.ts

```diff
@@ -1,5 +1,5 @@
 // Implementation of ErrorResponse.java in di-ipv-cri-lib
-class BaseError extends Error {
+export class BaseError extends Error {
```

Comment:
Ideally the BaseError should never be exported (in my opinion)
Instead, its a base class that's used by the other classes to create more specific errors relating to our system. I created the baseError so that we could add the relevant types and functions without code duplication.

Reply:
I have made the abstract class